translator/internaldata/metrics_to_oc.go

```diff
@@ -398,6 +444,23 @@ func labelValuesToOC(labels data.StringMap, labelKeys *labelKeys) []*ocmetrics.L
 		labelValue.HasValue = true
 	}
 
+	// Visit all defined labels in the MetricDescriptor and override defaults with actual values
```

Comment:
Shouldn't we do this before previous section? I assume if the same label is defined both in `MetricDescriptor` and in the particular data point then the label in data point should have the effect?